test/e2e/test_item.py

```diff
@@ -935,11 +965,44 @@ def test_partial_update_item(test_client):
     assert item == {**ITEM_POST_EXPECTED, **item_patch, "catalogue_item_id": catalogue_item_id, "system_id": system_id}
 
 
+def test_partial_update_item_usage_status(test_client):
+    """
+    Test updating an item with a invalid usage status
+    """
```

Comment:
I dont think this test is needed `test_partial_update_item` already changes the usage status. Also this is checking the error currently the same as above.